All Kind tests in CI failed consitently

**Describe the bug**
All kind tests failed because of same error: `docker image antrea/antrea-ubuntu:latest failed to load`.

```
kubectl cluster-info --context kind-kind

Not sure what to do next? 😅 Check out https://kind.sigs.k8s.io/docs/user/quick-start/
deployment.apps/coredns patched
Configuring networks
removing worker nodes kind-worker kind-worker2 from networks  bridge
disconnected worker kind-worker2 from network bridge
disconnected worker kind-worker from network bridge
creating network kind-0 with 20.20.20.0/24
connected worker kind-worker to network kind-0
node kind-worker is ready with ip change to 20.20.20.2 with gw 20.20.20.1
connected worker kind-worker2 to network kind-0
node kind-worker2 is ready with ip change to 20.20.20.3 with gw 20.20.20.1
current ip   InternalIP:  172.17.0.2, wait for new node ip 20.20.20.2
load images
loaded image busybox
loaded image nginx
docker image antrea/antrea-ubuntu:latest failed to load
 Test failed cleaning testbed
Deleting cluster "kind" ...
deleted networks kind-0
##[error]Process completed with exit code 1.
```

--- a/.github/workflows/kind-upgrade.yml
+++ b/.github/workflows/kind-upgrade.yml
@@ -28,6 +28,11 @@ jobs:
     needs: build-antrea-image
     runs-on: [ubuntu-18.04]
     steps:
+    - name: Free disk space
+      # https://github.com/actions/virtual-environments/issues/709
+      run: |
+        sudo apt-get clean
+        df -h
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v1
       with:
@@ -54,6 +59,11 @@ jobs:
     needs: build-antrea-image
     runs-on: [ubuntu-18.04]
     steps:
+    - name: Free disk space
+      # https://github.com/actions/virtual-environments/issues/709
+      run: |
+        sudo apt-get clean
+        df -h
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v1
       with:
@@ -80,6 +90,11 @@ jobs:
     needs: build-antrea-image
     runs-on: [ubuntu-18.04]
     steps:
+    - name: Free disk space
+      # https://github.com/actions/virtual-environments/issues/709
+      run: |
+        sudo apt-get clean
+        df -h
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v1
       with:

--- a/.github/workflows/kind.yml
+++ b/.github/workflows/kind.yml
@@ -28,6 +28,11 @@ jobs:
     needs: build-antrea-image
     runs-on: [ubuntu-18.04]
     steps:
+    - name: Free disk space
+      # https://github.com/actions/virtual-environments/issues/709
+      run: |
+        sudo apt-get clean
+        df -h
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v1
       with:
@@ -54,6 +59,11 @@ jobs:
     needs: build-antrea-image
     runs-on: [ubuntu-18.04]
     steps:
+    - name: Free disk space
+      # https://github.com/actions/virtual-environments/issues/709
+      run: |
+        sudo apt-get clean
+        df -h
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v1
       with:
@@ -80,6 +90,11 @@ jobs:
     needs: build-antrea-image
     runs-on: [ubuntu-18.04]
     steps:
+    - name: Free disk space
+      # https://github.com/actions/virtual-environments/issues/709
+      run: |
+        sudo apt-get clean
+        df -h
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v1
       with:
@@ -106,6 +121,11 @@ jobs:
     needs: build-antrea-image
     runs-on: [ubuntu-18.04]
     steps:
+    - name: Free disk space
+      # https://github.com/actions/virtual-environments/issues/709
+      run: |
+        sudo apt-get clean
+        df -h
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v1
       with:

